ex4B/ex4Spec.js

```diff
@@ -0,0 +1,12 @@
+var ex4 = require('./ex4.js');
+
+describe("toNumber", function() {
+
+	it("converte the chacter 7 & 1 string  to a number 71", function() {
```

Comment:
Please add a few more test cases: single digit (5), negative number (-3), decimal (0.6), negative and decimal (-3.5), many digits (12351231), etc. And their negatives: a5, 5g, 3-, --3, -3-, .4, 4.4.4, 4., aa, etc.